README.md

```diff
@@ -0,0 +1 @@
+# Survey-Form
```

Comment:
Hi @TekyRey

Great work on your survey form project 👏
You've done well using Gitlow and setting up the project, however, there are some issues i feel you need to improve on.

To highlight 🎯
Nice design. ✔️


Check the comments under the review for some improvements recommendations.